thermal.c

```diff
@@ -84,9 +84,17 @@ static ssize_t mwl_thermal_show_temp(struct device *dev,
 	}
 
 	temperature = priv->temperature;
+	if (priv->chip_type == MWL8864) {
+		/* unit is in fahrenheit for this chipset */
+		temperature -= 32;
+		temperature *= 10;
+		temperature /= 18;
+		ret = snprintf(buf, PAGE_SIZE, "%d\n", temperature * 1000);
+	} else {
+		ret = snprintf(buf, PAGE_SIZE, "%d\n", temperature * 100);
```

Comment:
Original code had `* 1000`, was this an intentional change?

Reply:
My question isn't about the conversion from fahrenheit to celcius, it's about switching other chips from multiplying by 1000 to multiplying by 100, so dividing the previously reported values by 10.

This part isn't really explained in the commit message, that apparently reported values were off by a magnitude.